_summaries/ramachandran2017searching_2.md

```diff
@@ -1,19 +1,63 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: ramachandran2017searching
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: liaojh1998
+score: 6
 ---
 
-TODO: Summarize the paper:
 * What is the core idea?
+
+    This paper proposed a reinforcement learning-based search method for finding good activation functions, then analyzed and evaluated the best found function, Swish, which is defined as $$x \cdot \sigma(\beta x)$$. According to evaluations, Swish is found to reliably outperform ReLU and multiple other activation functions in a wide variety of tasks and networks.
+
 * How is it realized (technically)?
+
+    * _Activation Function Search_
+
+        The authors restricted the search space to functions composed from functions that takes in 1 or 2 scalar values and output a scalar. This is done by composing from multiple repetitions of the "core unit":
+
+        <p align="center">
+            <img src="https://d3i71xaburhd42.cloudfront.net/c8c4ab59ac29973a00df4e5c8df3773a3c59995a/2-Figure1-1.png" width="30%" height="30%">
+        </p>
+
+        Note that the inputs of the unary functions are restricted to the preactivation $$x$$ and the binary function outputs. This way, the activation function can always take a scalar value as input and output a single scalar value, like ReLU.
```

Comment:
Maybe give a few examples of the unary or binary functions in the search space.

Reply:
Added all examples for reference.